src/js/data/names.js

```diff
@@ -3436,7 +3436,7 @@ let first = {
         ["A.C.", 5],
         ["A.J.", 63],
         ["A.W.", 64],
-        ["ATorri", 65],
```

Comment:
See line 3 of this file :)

I should probably do this part since I have the data. I'm not sure if it's correct in all cases though. I'm inclined to leave it as is. Like DAndre looks worse than D'Andre, right?